tests/kernel/mem_protect/userspace/prj.conf

```diff
@@ -1 +1,3 @@
 CONFIG_ZTEST=y
+CONFIG_INIT_STACKS=y
```

Comment:
I am wondering whether INIT_STACKS should be in all kernel tests, and not just in user-mode.

Reply:
It might be a reasonable default to turn on for all CONFIG_TEST targets, yes.
In this case, I enable it here so that stack_analyze() works

Reply:
I think we should just enable it in the tests. The more stuff we have enable in the tests, the best coverage we get ;)